gateway/src/apicast/balancer.lua

```diff
@@ -52,6 +52,20 @@ local function get_peer(balancer, upstream)
   return peer
 end
 
+local function set_timeouts(balancer, timeouts)
+  if not timeouts then return end
+
+  local _, err = balancer:set_timeouts(
+      timeouts.connect_timeout,
+      timeouts.send_timeout,
+      timeouts.read_timeout
+  )
+
+  if err then
+    ngx.log(ngx.WARN, 'Error while setting balancer timeouts: ', err)
```

Comment:
I read the resty balance timeout, and the error message is not too descriptive[0]. I would add the connect_timeout, send_timeout and read_timeout in the warning messages. 

https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/balancer.lua#L166-L206

Reply:
You're right, the error does not show the values. I added an `inspect()` there :+1: